src/Plugin/Drupal8/Audit/SettingAnalysis.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal8\Audit;
+
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\Audit\AbstractAnalysis;
+
+/**
+ * Retrieve all Drupal settings. Settings can be set in settings.php in the $settings array.
+ */
+class SettingAnalysis extends AbstractAnalysis
+{
+
+
+    public function configure()
```

Comment:
We can remove this method right?

Reply:
If the configure() method is removed, then Drutiny will fail because the `expression` parameter is not defined. I've found that I need to call `parent::configure()` in order to not have to re-define the AbstractAnalysis' params.

Reply:
This doesn't make sense. If the configure method is removed, then the parent configure method is called by default no?

Reply:
I have removed the configure() method and it is working as expected.